From e2f1d149c77fd84d709d84b8a568c369c7ed680c Mon Sep 17 00:00:00 2001
From: "wenping.zhang" <wenping.zhang@rock-chips.com>
Date: Fri, 21 Oct 2016 18:28:45 +0800
Subject: [PATCH] arm64: dts: rockchip: update the dts for excavator discrete
 vr device.

Change the configs for RAYKEN 5.46' lcd which is defaultly used for
discrete vr lcd.

Change-Id: I3894697367229ea059b9200fd2ad5aac8781b7df
Signed-off-by: wenping.zhang <wenping.zhang@rock-chips.com>
---
 .../boot/dts/rockchip/rk3399-sapphire-excavator-disvr.dts    | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-disvr.dts b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-disvr.dts
index 60c1dd19ec50..afead637cdc4 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-disvr.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-disvr.dts
@@ -116,13 +116,12 @@
 &vopb_rk_fb {
 	status = "okay";
 	rockchip,prop = <EXTEND>;
-	rockchip,dsp_mode = <ONE_VOP_DUAL_MIPI_HOR_SCAN>;
 };
 
 &disp_timings {
 	timing0 {
-		screen-width = <120>;
-		screen-hight = <68>;
+		screen-width = <68>;
+		screen-hight = <120>;
 	};
 };
 
-- 
2.35.3

